sdk/storage/Azure.Storage.Common/src/Shared/StorageExtensions.cs

```diff
@@ -59,5 +60,14 @@ public static string GenerateBlockId(long offset)
             BitConverter.GetBytes(offset).CopyTo(id, 0);
             return Convert.ToBase64String(id);
         }
+
+        public static async Task<AccessToken> GetCopyAuthorizationTokenAsync(
```

Comment:
Do we still need this?

Reply:
Yup this is being used in the `BlobBaseClient.GetCopyAuthorizationTokenAsync`